src/items/fish/raw-anchovies.js

```diff
@@ -7,6 +7,9 @@ export default class RawAnchovies extends RawFish {
 
     // Other
     cost = 15;
+    cookingXp = 30;
+    cookingLvl = 1;
+    burnLvlLimit = 34;
```

Comment:
Didn't know that's how they did it 🤔

Reply:
I'm not sure if my equation is right, but there is definitely a limit